### PR TITLE
DAG-1662: Create documentation site

### PR DESCRIPTION
**Pull request by dbradf**

### CODE REVIEW

File: README.md
Comment (dbradf) [inline README.md:40]: I already configured the github pages, so this link is live if you want to take a look.

File: emm-docs/content/getting-started/_index.md
Comment (dbradf) [inline emm-docs/content/getting-started/_index.md:2]: Note: `weight` lets you configure the order in which the pages will appears in the navigation.

File: emm-docs/themes/hugo-geekdoc/LICENSE
Comment (dbradf) [inline emm-docs/themes/hugo-geekdoc/LICENSE:1]: You can pretty much ignore everything under the `hugo-geekdoc` directory. It is a theme that was downloaded and extracted from [here](https://github.com/thegeeklab/hugo-geekdoc).

File: emm-docs/content/getting-started/global-options.md
Comment (vincivans) [inline emm-docs/content/getting-started/global-options.md:13]: [nit] Those

Comment (vincivans) [inline emm-docs/content/getting-started/global-options.md:13]: [nit] Those can be provided...

File: emm-docs/content/subcommands/commit-queue.md
Comment (vincivans) [inline emm-docs/content/subcommands/commit-queue.md:16]: [nit] and should not be included.

File: emm-docs/content/subcommands/list-modules.md
Comment (vincivans) [inline emm-docs/content/subcommands/list-modules.md:33]: It seems the demo is dead on the website

File: emm-docs/content/subcommands/patch.md
Comment (vincivans) [inline emm-docs/content/subcommands/patch.md:16]: [nit] same as before

**Review by vincivans — APPROVED** [review]
Looks pretty good overall! It makes a nice interactive page for the user. Just some nits need to be addressed before merge.

File: emm-docs/content/subcommands/commit-queue.md
Comment (dbradf) [reply emm-docs/content/subcommands/commit-queue.md]: fixed.

File: emm-docs/content/subcommands/list-modules.md
Comment (dbradf) [reply emm-docs/content/subcommands/list-modules.md]: I've removed this for now.

File: emm-docs/content/subcommands/patch.md
Comment (dbradf) [reply emm-docs/content/subcommands/patch.md]: fixed.

File: CHANGELOG.md
Comment (dbradf) [reply CHANGELOG.md:1]: This is a limitation with how we are including the changelog. Hugo automatically adds a title, but we also include one in the file itself. There is likely a way to resolve this, but since it is not a big problem, I'm going to hold off on it for now.

**Comment by dbradf** [comment]
evergreen merge

**Pull request merged**
